Source/WebCore/rendering/svg/RenderSVGInline.cpp

```diff
@@ -142,11 +142,17 @@ void RenderSVGInline::willBeDestroyed()
 void RenderSVGInline::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle)
 {
 #if ENABLE(LAYER_BASED_SVG_ENGINE)
-    if (!document().settings().layerBasedSVGEngineEnabled() && diff == StyleDifference::Layout)
+    if (!document().settings().layerBasedSVGEngineEnabled() && diff == StyleDifference::Layout) {
         setNeedsBoundariesUpdate();
+        if (style().hasTransform())
```

Comment:
What about the case when the transition is from a style with transform to a style without it?

Reply:
So (!style && style().hasTransform), this you mean?

Reply:
I would have expected to see:

`oldStyle && oldStyle->hasTransform()`

But what really matters is test cases to confirm this is needed.